### DISCUSSION TITLE
Migrate only a v0.5 SQLite database?

### DISCUSSION THREAD

**Discussion by riborg** [mealie-recipes/mealie#3067] [General]
Is it possible to save a 0.5.0-database?

I am running Watchtower in my container environment and as soon as the new version was available, it downloaded and restarted my Mealie. Everything seems to be gone.

I can see that the old `mealie_v0.5.0.db` is still present in my data folder. Is there any way I can do the migration afterwards?

I tried to start the 0.5.x image again but I can't login to make a backup.

Help..

**Comment by dasrot** [comment]
I had the same issue, mealie 1.0 did not migrate the old database.  I ended up copying my data over to a new folder, renaming mealie_v0.5.0.db to mealie.db in the new folder (not sure if this is necessary) and then created a v0.5.6 container with that folder and was able to get into my old database.  I then backed it up and loaded it into the v1.0.1 instance per the instructions.

**Reply by riborg** [reply]
Thanks, I finally managed to do the same. I realized that I was accessing the wrong web port.. Apparently that was changed as well. :)
